### ISSUE TITLE
Missing files

### ISSUE THREAD

**Issue by tommygebru** [Ayomiobasan/f1-3-c2p1-colmar-academy#1]
Hey there,
My name is Tommy and I work on the Project Review team. I was attempting to review your project, but was not able to, because your CSS & HTML are missing; therefore, there is no viewable webpage or project in the browser.

In order to receive a review, please resubmit your project by following these steps:
Navigate to the project submission page for the project you submitted in the Codecademy Dashboard
Click the “Submit” panel
Click “Resubmit my project”
Once you’ve completed your project and uploaded it to this repo, you will be able to receive a review. To do so, please notify Customer Support by submitting a request via this form.
Fill out the form using the following template:
```
*Subject*: Request for Project Review

*Description*:

I am submitting a project for review as there was an error with my previous submission. 

Please see information below:

Intensive Name:
Cohort Start Date:
Project Title:
Git Repo Link:
```

Thanks,
Codecademy Coaches

**Comment by Ayomiobasan** [comment]
Okay, I submitted and deleted the wrong folder

**Comment by tommygebru** [comment]
Very cool Ayomiobasan, looking forward to reviewing your project **Colmar**!

Also were you able to submit your project **Dasmotos** on time? I dont see it in your list of github repositories 👀

**Comment by Ayomiobasan** [comment]
no, the network in my area and electricity here is very bad.. so it really
affect me working with my system
